views/index.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<!--[if IE 9]>
+<html class="lt-ie10" lang="en" >
+<![endif]-->
+
+<head>
+
+
+
+</head>
+
+
+
+
+
```

Comment:
This file should only have the content specific to the template. The `<head>` tag etc is supposed to be placed in the `layout´ file